app/db/crud_event.py

```
@@ -0,0 +1,29 @@
+    del event_dict['date']
+    del event_dict['time']
+    event_dict['created_at'] = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+    insert_query = f"""
+    INSERT INTO events {str(tuple(event_dict.keys())).replace("'","")} VALUES {tuple(event_dict.values())}
+    """
+    conn = initialize_db()
+
+    try:
+        cur = conn.cursor()
+        cur.execute(insert_query)
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)